redwood/.env.defaults

```diff
@@ -3,7 +3,7 @@
 # system. Any custom values should go in .env and .env should *not* be checked
 # into version control.
 
-CHAIN_DEPLOYMENT=production
```

Comment:
This seems like the better default to me, and we can override it in Render. Any reason no to do this?

Reply:
as in, the better default is not to specify a CHAIN_DEPLOYMENT at all? That sounds like it could work. I was thinking safest to default to production so we don't accidentally release something in dev mode... But even better might be to just throw an error due to lack of the env variable, which is probably what we do if there is no default.

Reply:
Actually most apps I've worked with default to dev if no environment is specified, which seems like a good default to me.

I removed the default entirely here because I was (incorrectly) worried that if we somehow forget to set the `CHAIN_DEPLOYMENT` when we deploy the production app it would point at some random dev contract and confuse people. However, that's not actually an issue, because we don't check in the dev contract addresses, so the app would just fail to sync.

You ok with defaulting to development here?